src/translations/screens/Ticketing.ts

```diff
@@ -47,8 +47,12 @@ const TicketingTexts = {
     },
   },
   activeFareProductsAndReservationsTab: {
-    label: _('Aktive', 'Active', 'Aktive'),
-    a11yLabel: _('Aktive billetter', 'Active tickets', 'Aktive billettar'),
+    label: _('Mine billetter', 'My tickets', 'Mine billettar'),
+    a11yLabel: _(
+      'Mine aktive billetter',
+      'My active tickets',
+      'Mine aktive billettar',
+    ),
```

Comment:
Accessibility text should be the same as shown text, and there is no meta-info (like image or anything) that signals extra info regarding that the tab is a "My active tickets" tab. And this probably means we don't need to set an `accessibilityLabel` on the component at all.

Reply:
Will remove it on the "my tickets" tab. There is also a similar case on the "buy" tab, where it in the a11y labels says "buy tickets". Here I belive it should be kept as just the word "buy" is not very describing in itself. Agree or not? @gorandalum

Reply:
was decided by @hannahATB to keep it as "buy"